packages/idyll-components/src/scroller.js

```diff
@@ -52,9 +52,11 @@ class Scroller extends React.Component {
       .setup({
         step: `#idyll-scroll-${this.id} .idyll-step`, // required
         container: `#idyll-scroll-${this.id}`, // required (for sticky)
-        graphic: `#idyll-scroll-${this.id} .idyll-scroll-graphic` // required (for sticky)
+        graphic: `#idyll-scroll-${this.id} .idyll-scroll-graphic`, // required (for sticky)
+        progress: this.props.progress != undefined ? true : false
```

Comment:
Small thing, but the project style is to always prefer `!==` over `!=` unless there's a specific reason why it isn't appropriate.

Reply:
Sure. Thanks for pointing out